diff --git a/linux/installer/bin/install-sgx-psw.bin.tmpl b/linux/installer/bin/install-sgx-psw.bin.tmpl
index 10a8952..405a4cb 100755
--- a/linux/installer/bin/install-sgx-psw.bin.tmpl
+++ b/linux/installer/bin/install-sgx-psw.bin.tmpl
@@ -30,31 +30,10 @@
 #
 #
 
-grep intel_sgx /lib/modules/$(uname -r)/modules.builtin &> /dev/null
-if [[ $? != "0" ]]; then
-    modinfo isgx &> /dev/null
-    if [[ $? != "0" ]]; then
-        lsmod | grep isgx &> /dev/null
-        if [[ $? = "0" ]]; then
-            echo "Warning: You did not follow the document to install the driver package"
-            echo
-        else
-            echo "Please install the driver package before the PSW package"
-            exit 4
-        fi
-    fi
-fi
-
 set -e
 
 PKG_NAME="Intel SGX PSW Package"
 
-if test $(id -u) -ne 0; then
-    echo "Root privilege is required to install $PKG_NAME."
-    exit 4
-fi
-
-
 PKG_ARCH=@arch@
 ARCH=$(uname -m)
 
